src/btree/bt_sync.c

```diff
@@ -396,7 +467,7 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
     WT_TXN *txn;
     uint64_t internal_bytes, internal_pages, leaf_bytes, leaf_pages;
     uint64_t oldest_id, saved_pinned_id, time_start, time_stop;
-    uint32_t flags, rec_flags;
+    uint32_t flags, load_throttle, rec_flags;
```

Comment:
`load_throttle` can just be a `u_int`, or even `int`

Reply:
Fixed in 2fbb617ec (and obsoleted in 395664db3)